Sources/Sentry/SentryANRTrackingIntegration.m

```diff
@@ -70,19 +63,32 @@ - (BOOL)shouldBeDisabled:(SentryOptions *)options
 
 - (void)uninstall
 {
-    [self.tracker stop];
+    [self.tracker removeListener:self];
 }
 
 - (void)anrDetected
 {
-    [self.appStateManager
-        updateAppState:^(SentryAppState *appState) { appState.isANROngoing = YES; }];
+    NSString *message =
+        [NSString stringWithFormat:@"Application Not Responding for at least %li ms.",
+                  (long)(self.options.anrTimeoutInterval * 1000)];
+
+    NSArray<SentryThread *> *threads =
+        [SentrySDK.currentHub.getClient.threadInspector getCurrentThreadsWithStackTrace:YES];
+
+    SentryEvent *event = [[SentryEvent alloc] initWithLevel:kSentryLevelError];
+    SentryException *sentryException = [[SentryException alloc] initWithValue:message
+                                                                         type:@"App Hanging"];
+    sentryException.mechanism = [[SentryMechanism alloc] initWithType:@"anr"];
+
+    event.exceptions = @[ sentryException ];
+    event.threads = threads;
+
+    [SentrySDK captureEvent:event];
```

Comment:
`h`: @marandaneto, can you have a look if creating the ANR event aligns with Android, please?